opcua/ua/uaprotocol_auto.py

```diff
@@ -9948,7 +9948,7 @@ def __init__(self):
         self._freeze = True
 
     def __str__(self):
-        return 'PublishResponse(TypeId:{self.TypeId}, ResponseHeader:{self.ResponseHeader}, Parameters:{self.Parameters})'
+        return 'PublishResponse(TypeId:{}, ResponseHeader:{}, Parameters:{})'.format(self.TypeId, self.ResponseHeader, self.Parameters)
```

Comment:
I just happened to update a few of these while debugging. This is out of scope but might as well keep the changes.